docker/server/init-defaults.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
```

Comment:
WHy don't you put that into https://github.com/ClickHouse/ClickHouse/blob/615104c8202adc2b5b15977b65d426ce0b9f57ed/docker/server/entrypoint.sh ? 

I think it't better to keep all the entypoint logic together.

Reply:
Code moved